lib/puppet/provider/network_snmp/cisco_nexus.rb

```diff
@@ -0,0 +1,46 @@
+require 'cisco_node_utils'
+
+# Implementation for the network_snmp type using the Resource API.
+class Puppet::Provider::NetworkSnmp::CiscoNexus
+  def get(_context)
+    @network_snmp ||= Cisco::SnmpServer.new
+
+    current_state = {
+      name:     'default',
+      enable:   @network_snmp.protocol? ? true : false,
+      contact:  @network_snmp.contact.empty? ? 'unset' : @network_snmp.contact,
+      location: @network_snmp.location.empty? ? 'unset' : @network_snmp.location,
+    }
+
+    [current_state]
+  end
+
+  def set(context, changes)
+    changes.each do |name, change|
```

Comment:
I know it is minor, but would it be possible to get a simple unit test to get the coverage for this function?